decidim/decidim
# Merge DummyResource and Dev namespaces

#### :tophat: What? Why?
While working to release metaDecidim, we have noticed there are some issues with the factoryBot DummyResource, which was fixed by #12123. While upgrading another application to Decidim 0.28, i have started to see there are a lot of errors like the one in the below image (the same error fixed by #12123). Upon investigation, i have noticed the 2 applications had the rails config "config.load_defaults" different. MetaDecidim used `config.load_defaults 5.1`, the faulty application used `config.load_defaults 6.1`. 

In order to validate my assumption, i have changed the same config in MetaDecidim, and started to see the same error. I have changed the MetaDecidim gem source to my decidim local folder, and i have started working on the fix. When the implementation was finished, the MetaDecidim test suite was successful. 

![image](https://github.com/decidim/decidim/assets/105683/55a59d96-7860-433f-b446-73926ea32d2e)

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #12123
- Related to #10682

#### Testing
1. Download Metadecidim and install decidim-0.28.rc4 
2. Run test suite and see is successful 
3. Change `config.load_defaults 6.1` in `config/application.rb`, repeat step 2 
4. See the testsuite is failing
5. Apply patch  
6. Repeat step 3 
7. See test suite is successful

:hearts: Thank you!
